bundles/config/org.eclipse.smarthome.config.discovery.test/src/test/groovy/org/eclipse/smarthome/config/setup/test/inbox/DynamicThingUpdateOSGITest.groovy

```diff
@@ -179,4 +184,40 @@ class DynamicThingUpdateOSGITest extends OSGiTest {
 
         unregisterService(thingHandlerFactory)
     }
+
+    @Test
```

Comment:
But the persistentInbox does not mock the thing handler nor the thing handler factory. I just wanted to reuse the existing test infrastructure. Why not just renaming the test class?